packages/components/src/mobile/inserter-button/index.native.js

```diff
@@ -88,7 +88,9 @@ class MenuItem extends Component {
 							/>
 						</View>
 					</View>
-					<Text style={ modalItemLabelStyle }>{ blockTitle }</Text>
+					<Text numberOfLines={ 3 } style={ modalItemLabelStyle }>
+						{ blockTitle }
+					</Text>
```

Comment:
I set the maximum number of lines for the block's title to three, this will prevent rows to be too big but enough for display long texts.

Before|After
--|--
<img src=https://user-images.githubusercontent.com/14905380/117287365-8d943980-ae6a-11eb-8b8c-02a1a7c067d8.png width=500>|<img src=https://user-images.githubusercontent.com/14905380/117287376-908f2a00-ae6a-11eb-8350-e8ec4cd7eeef.png width=500>